src/plugins.js

```diff
@@ -11,8 +11,8 @@ const redirectsPlugin = [
           '/essentials',
           '/essentials/overview'
         ],
-        to: '/what-is-macrometa'
-      },
+        to: '/quickstart'
+      },q
```

Comment:
There's a stray `q` here, causing the latest issue.

Reply:
Need a facepalm emoji...